dietpi/func/dietpi-banner

```diff
@@ -48,7 +48,19 @@
 	# Set defaults
 	aENABLED=(1 0 1 0 0 1 0 0 0 0 0 1)
 
-	GREEN_LINE=' \e[38;5;154m─────────────────────────────────────────────────────\e[0m'
+	aCOLOUR=(
+
+		'\e[0m' #Reset
```

Comment:
@Fourdee 
Perhaps `\e[0m` should stay hard corded to not allow accidental colour mess?